lua/downloader/cache_load.lua

```diff
@@ -19,6 +18,17 @@ function MODULE:Run(context)
 
             context.ignoreResources[addon.wsid] = true
             context.gamemodeAddons[addon.wsid] = scanned.isGamemode
+            context.manualAddons[addon.wsid] = scanned.manual
+
+            if scanned.scanResult then -- Compatibility with older caches
+                context.scanResult[addon.wsid] = scanned.scanResult
+            else
```

Comment:
why we should cache scanResult? isn't this if sufficient?

Reply:
Gamemodes and resources scanning are blocked after they are cached, so the scan results need to be loaded from storage or rebuilt. In fact we can choose between these, and I'll remove the data from the cache to keep things simple.

Btw you accidentally found a bug, because I forgot to rebuild the gamemodes status... They're all being classified as Lua.

Reply:
ed289f83a0979a55411f663074fc0199cb97b953